src/test/java/com/microsoft/sqlserver/jdbc/bulkCopy/ISQLServerBulkRecordIssuesTest.java

```diff
@@ -253,6 +256,26 @@ public void afterEachTests() throws SQLException {
     }
 }
 
+
```

Comment:
Why is this in a different file?

Reply:
This interface already has another (very similar) bulk data class used for test, so I think it makes sense to keep this code here.

Reply:
That class is only used inside the same test file. I would keep  the required classes for a test inside its own file.

Reply:
Ok, I've updated the test.